### ISSUE TITLE
ACC-253.1 (Level AA) _ Modify Modules _ Edit field _ No visible label provided 

### ISSUE THREAD

**Issue by shubha1090** [kyma-project/busola#3125]
1.1 Internal Message Details
Internal Message:	
Test Plan:	ACC SAP Kyma runtime 2024
Message Priority:	High
Affected UI	Edit field
Affected Check Point:	ACC-253.1
Short text:	[ACC-253.1 (Level AA) _ Modify Modules _ Edit field _ No visible label provided.docx](https://github.com/user-attachments/files/16611744/ACC-253.1.Level.AA._.Modify.Modules._.Edit.field._.No.visible.label.provided.docx)

1.2 Application Details
Application Name	Kyma-dashboard
Technology:	SAP UI5 
Component:	
URL:	https://dashboard.stage.kyma.cloud.sap/cluster/garden-kyma-stage--c-7074971-external/kymamodules
Username/Password:	NA
1.3 Test Environment Details
Web browser:	Google Chrome Version 127.0.6533.89
Operating System:	Microsoft Windows 11 x64
Screen Reader:	JAWS 2024 Version 2024.2406.121 ILM

1.4 JIRA/SNOW Details
Issue:	Bug
Labels:	Accessibility

2 Screen Flow
•	Launch the application using given URL. 
•	Select the ‘Link to Kyma dashboard’ from the Kyma Environment section on your SAP BTP subaccount page. Now Select ‘Modify Modules’ button which opens ‘Modules’ page.
•	Open JAWS 2024 screen reader and turn on VPC mode.
•	Navigate to the ‘Search’ edit field from the main region of ‘Modules’ page.

3. Issue Type: Manual
Observed Behavior:
There is no visible label provided for the ‘Search’ edit field from the main region of ‘Modules’ page which describes the purpose of the UI.
Screenshot:
 

Expected Behavior:
An appropriate and meaningful visible label provided for the ‘Search’ edit field  from the main region of ‘Modules’ page which describes the purpose of the UI as the placeholder value cannot be considered a label since it disappears as the user types some text .

Note: 
This issue exists across the application and thus fix should be provided for all the occurrences.

________________________________________________________________________
Kindly refer to attached error and issue processing document for more details.
Regards,
Shubha

**Comment by shubha1090** [comment]
[ACC-253.1 (Level AA) _ Modify Modules _ Edit field _ No visible label provided.docx](https://github.com/user-attachments/files/16611825/ACC-253.1.Level.AA._.Modify.Modules._.Edit.field._.No.visible.label.provided.docx)
